src/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.Unescaping.cs

```diff
@@ -10,6 +10,31 @@ namespace System.Text.Json
 {
     internal static partial class JsonReaderHelper
     {
+        public static bool TryGetUnescapedBase64Bytes(ReadOnlySpan<byte> utf8Source, int idx, out byte[] bytes)
+        {
+            byte[] unescapedArray = null;
+
+            Span<byte> utf8Unescaped = utf8Source.Length <= JsonConstants.StackallocThreshold ?
+                stackalloc byte[utf8Source.Length] :
```

Comment:
I would like to validate first/see the difference before making the change. Also, I would like to do so all up, rather than piecemeal (in a separate PR), since the current pattern is used everywhere in `S.T.Json` atm, and for now I want to remain consistent.